packages/app/src/systems/Settings/machines/settingsMachine.ts

```diff
@@ -34,6 +32,9 @@ export const settingsMachine = createMachine(
     predictableActionArguments: true,
     id: 'settingsMachine',
     initial: 'idle',
+    context: {
+      words: [],
```

Comment:
do we store the mnemonic in the settingsMachine context?  How safe is this?

Reply:
GQ! This is only on memory during the time you are seeing it. After you close the popup, the context is reset.